platform/macos/CHANGELOG.md

```diff
@@ -1,8 +1,34 @@
 # Changelog for Mapbox Maps SDK for macOS
 
+ ## 0.7.2 - June 22, 2018
+
 * Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```

Comment:
And here - should match the one under the 0.8.0 section

Reply:
same as above